src/common-models/month-info.model.ts

```diff
@@ -25,3 +30,47 @@ export interface MonthInfoModel {
   frozen_shifts: [number | string, number][];
   dates: number[];
 }
+
+export function validateMonthInfo(
+  monthInfo: MonthInfoModel,
+  containerType: ScheduleContainerType
+): void {
+  const scheduleLen = monthInfo.dates.length;
+  if (!SCHEDULE_CONTAINERS_LENGTH[containerType].includes(scheduleLen)) {
```

Comment:
Mi się wydaje, że my powinniśmy podzielić to na dwie funkcje. 
`validateMonthInfoModel`
`validateScheduleInfoModel`

Na razie może być tak, że pod spodem będą korzystać z tego samego, ale tak naprawdę nie mamy gwarancji tego, że `monthInfoModel` zawsze będzie wyglądać tak samo jak `scheduleInfoModel`

Reply:
Ok, będzie bezpieczniej